app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/InAppUpdateHelper.kt

```diff
@@ -0,0 +1,60 @@
+package cz.covid19cz.erouska.exposurenotifications
+
+import android.app.Activity
+import android.content.Context
+import android.content.IntentSender
+import com.google.android.play.core.appupdate.AppUpdateManagerFactory
+import com.google.android.play.core.install.model.AppUpdateType
+import com.google.android.play.core.install.model.UpdateAvailability
+import cz.covid19cz.erouska.AppConfig
+import cz.covid19cz.erouska.BuildConfig
+import cz.covid19cz.erouska.utils.L
+
+object InAppUpdateHelper {
+
+    const val APP_FORCE_UPDATE_REQUEST_CODE = 1777
+
+    fun isObsolete(): Boolean {
+        return BuildConfig.VERSION_CODE < AppConfig.minSupportedVersionCodeAndroid
+    }
+
+    /**
+     * Check for update and once it's available show force update dialog.
+     * Check for soft (flexible) update by default.
+     */
+    fun checkForAppUpdateAndUpdate(activity: Activity, updateType: Int = AppUpdateType.FLEXIBLE) {
```

Comment:
Ale ten Flexible typ se nikdy nepoužívá? Vyhodil bych ten parametr, pokud to nepoužíváme, mate to.

Reply:
Na jednom standupu jsme se domlouvali, že naintegrujeme soft-update, tak jsem to na to připravil

Reply:
Nemám rád kód "pro budoucnost", často se stane že se to pak nepoužije a smrdí to tam navždy. Ale je to spíš NIT.